src/org/exist/dom/persistent/QNamePool.java

```diff
@@ -113,7 +113,7 @@ private void clear() {
         items = 0;
     }
 
-    private QName insert(final QName value) throws HashtableOverflowException {
+    private QName insert(final QName value) throws HashSetOverflowException {
```

Comment:
@shabanovd I would not generally use these, and would actually like to move away from them to more standardised collections